_docs/operations/maintenance.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: guide
+title: StorageOS Docs - Maintenance
+anchor: concepts
+module: concepts/maintenance
+---
+
+# Maintenance
+
+## Node Maintenance
+
+## Draining a node
+
+Draining a node reschedules volumes to other nodes and marks the node as
+unschedulable.
+
+* If the volume does not have any replicas, a new replica will be created on a
+different node and promoted to master. The previous master will be removed.
+
+* If the volume has replicas, one will be promoted to master. The previous master
+will become a replica and be relocated to a different node.
+
+* If there are not enough available nodes, StorageOS will keep trying to evict all
+volumes while the node is in the drained state. Once a new node is added to the
+cluster, the volume will be moved automatically.
+
+Performing a node drain will not remove the StorageOS mounts living in that
+node. Any volume mounted in that specific node will be evicted but still hold
+the StorageOS mount making the data transparently available to the client, with
+zero downtime.
+
+To drain a node you can use the GUI or the StorageOS cli
+```bash
+$ storageos node drain node01
+node01
+```
+
+## Cordoning a node
+
+Cordoning a node marks the node as unschedulable without rescheduling any
+volumes running on the node. New volumes are unable to be scheduled nor can replicas
+be promoted on cordoned nodes.
+
+To cordon a node you can use the GUI or the StorageOS cli
+```bash
+$ storageos node cordon node01
+node01
+```
+
+## Cluster Maintenance Mode
+
+Freezing a StorageOS cluster means that no volumes will be moved, or replicas
+promoted while the cluster is frozen. This is useful behaviour during update
+operations. 
```

Comment:
We can mention that StorageOS decisions made based on node health will be ignored while cluster freeze.